src/public/js/rs.js

```diff
@@ -23,6 +40,10 @@ var RS = (function() {
   // The largest order coefficient is the [0] element of msg so as
   // to maintain the ordering of [msg, code symbols] in the coded msg.
   function encodeRSBlock(msg, n, k) {
```

Comment:
Mostly because we haven't entirely decided what we want the whole system to look like. I left it here in case we decide we want to use it at some point. Easy to switch either way so I think I will leave it for now unless you'd like to send these arguments off on their way to backspace oblivion.